python/guide/module/autograd.md

```diff
@@ -1,178 +1,260 @@
 # Automatic Differentiation
 
-<!-- adapted from diveintodeeplearning -->
-
-In
-machine learning, we *train* models to get better and better as a function of
-experience. Usually, *getting better* means minimizing a *loss function*, i.e. a
-score that answers "how *bad* is our model?" With neural networks, we choose
-loss functions to be differentiable with respect to our parameters. Put simply,
-this means that for each of the model's parameters, we can determine how much
-*increasing* or *decreasing* it might affect the loss. While the calculations
-are straightforward, for complex models, working it out by hand can be a pain
-(and often error-prone).
-
-The autograd package expedites this work by
-automatically calculating derivatives. And while most other libraries require
-that we compile a symbolic graph to take automatic derivatives, `autograd`
-allows you to take derivatives while writing  ordinary imperative code. Every
-time you make pass through your model, `autograd` builds a graph on the fly,
-through which it can immediately backpropagate gradients.
+### Why do we need to calculate gradients?
+
+Gradients are fundamental to the process of training neural networks, and tell us how to change the parameters of the network to improve its performance.
+
+<p align="center"><img src="./imgs/autograd_gradient.png" alt="drawing" width="400"/></p>
+
+Long Answer:
+
+Under the hood, neural networks are composed of operators (e.g. convolutions) which use parameters (e.g. weights in convolution kernels) for their computation, and it's our job to find the optimal values for these parameters. Gradients lead us to the solution!
```

Comment:
this may not be so intuitive to new readers. 

neural networks make use of operators (e.g. convolutions ... <insert another example), ...) which operate on parameters (also known as weights). The operators are fixed but the parameters are not and its our job to find optimal values for these parameters.

Reply:
done. although operators might not be fixed with dynamic graphs.